### PR TITLE
#213 gradle-docker-plugin is not supported 'HostConfig' settings (add restartPolicy parameter)

### PR DESCRIPTION
**Pull request by Nekitos**

### CODE REVIEW

File: src/main/groovy/com/bmuschko/gradle/docker/tasks/container/DockerCreateContainer.groovy [hunk 1]
```diff
@@ -167,6 +171,14 @@ class DockerCreateContainer extends AbstractDockerRemoteApiTask {
         exposedPorts << new ExposedPort(internetProtocol, ports)
     }
 
+    void setRestartPolicy(String name, int maximumRetryCount) {
+        restartPolicy = "${name}:${maximumRetryCount}"
+    }
+
+    void setRestartPolicy(String serialized) {
```
Comment (cdancy) [inline src/main/groovy/com/bmuschko/gradle/docker/tasks/container/DockerCreateContainer.groovy]: Lets remove this method. If folks want to specify the restartPolicy directly they can do so (restartPolicy = "random:123"). No need to have an extra method.

**Comment by cdancy** [comment]
@Nekitos good start. Left a couple comments. Just a reminder: you will need to provide a functional test before I will merge.

**Comment by Nekitos** [comment]
@cdancy  got it. Will do it asap. Thanks for reviewing.

**Comment by cdancy** [comment]
FYI: @Nekitos I'll be releasing a new version this weekend (probably Sunday night) if you want to get this change in.

**Pull request merged**